notes/src/App.js

```diff
@@ -56,6 +56,42 @@ class App extends Component {
                 return <NoteList notes={this.state.notes} />;
               }} />
           </Col>
+          <Col sm="3">
+            <Route path="/"
+              render={props => {
+                return <NoteList notes={this.state.notes} />;
```

Comment:
Your Route looks great! But isn't the NoteList component a list of all your notes? If so, why is it included more than once?